README.md

```diff
@@ -1 +1,13 @@
-# bitrise-add-new-project
\ No newline at end of file
+# bitrise-add-new-project
+
+Creates new project on Bitrise.io using a local existing bitrise.yml or running the Bitrise scanner.
+
+## one-liner
+
+### Create a Bitrise project under an organization
+
+`bash <(curl -sfL https://raw.githubusercontent.com/bitrise-io/bitrise-add-new-project/installer/_scripts/install.sh) --version "0.2.1" --api-token "<Bitrise personal access token>" --org "<organisation slug>" --public "<true|false>"`
```

Comment:
For me, including the version in the bash command makes our process a little more difficult. Either we need to keep it up-to-date or the website team need to be as fast as possible if there is a problem with a given version.

The reason I suggest to have always latest in the run script is because then we can manage versions almost instantly, moreover we are expecting all of our releases as tested properly, without any issue. In case if there is a very poor possibility to make something wrong then this way we have an instant option to get back to the previous version by deleting the release from github.